src/style.css

```diff
@@ -0,0 +1,107 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir1;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 500;
+}
+
+html {
+  margin-top: 0;
+}
+
+.form {
+  height: 70px;
+  position: relative;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  box-sizing: border-box;
+}
+
+.search {
+  color: #3d4e61;
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  padding: 23px 62px;
+  font-weight: 300;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  line-height: 22px;
+  font-size: 16px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+}
+
+.photo {
+  width: 19px;
+  bottom: 26px;
+  left: 26px;
+  position: absolute;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+}
+
+.search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-family: Avenir1, sans-serif ;
+  font-size: 16;
+  outline: none;
+  border: 1px solid #e1e7ed;
+}
+
+.form-small {
+  position: relative;
+  height: 42px;
+  box-sizing: border-box;
+}
+
+.search-small {
+  padding: 10px 33px;
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  line-height: 20px;
+  font-size: 14px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.photo-small {
+  width: 11px;
+  bottom: 16px;
+  left: 13px;
+  position: absolute;
+}
+
+.search-small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+}
+
+.search-small:focus {
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-family: Avenir1, sans-serif ;
+  outline: none;
+}
```

Comment:
Omg, you duplicate a log of styles. Fix this. [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2nd point